tools/proofers/preview.js

```diff
@@ -544,29 +539,20 @@ window.addEventListener('DOMContentLoaded', function() {
             }
         }
 
-        // heading should not be entirely bold
-        function boldHeading() {
-
-            function checkHead(type, text, block) {
-                // test for all bold
-                if((txt.indexOf("<b>", block.start) === block.start) && (txt.indexOf("</b>", block.start) === (block.end - 4))) {
-                    reportIssue(block.start, 3, type);
-                }
-            }
-
-            function testBlock(block) {
-                switch(block.type) {
-                case blockType.HEAD:
-                case blockType.SECHEAD:
-                case blockType.SUBHEAD:
-                    checkHead("noBold", txt, block);
-                    break;
-                default:
-                    break;
+        function testBoldBlock(block) {
+            if((txt.indexOf("<b>", block.start) === block.start) && (txt.indexOf("</b>", block.start) === (block.end - 4))) {
```

Comment:
the indices we see (`- 4`, `+ 3`, `+ 2`) could be easier if we explained them some, or computed them from a fixed value? Like:
```
if((txt.indexOf("<b>", block.start) === block.start) &&
   (txt.indexOf("</b>", block.start) === (block.end - "</b>".length))) {
```
obviously subjective, just thinking out loud.

Reply:
That seems like a good idea.